server/fleet/hosts.go

```diff
@@ -913,6 +913,7 @@ type HostMDM struct {
 	MDMID                  *uint   `db:"mdm_id" json:"-" csv:"-"`
 	Name                   string  `db:"name" json:"-" csv:"-"`
 	DEPProfileAssignStatus *string `db:"dep_profile_assign_status" json:"-" csv:"-"`
+	AwaitingConfiguration  bool    `db:"awaiting_configuration" json:"-"`
```

Comment:
I think this is not needed, right? We opted not to change the `GetHostMDM` query?